usrp_drivers/n200/n200_driver.cpp

```diff
@@ -540,19 +553,26 @@ void receive(zmq::context_t &driver_c, USRP &usrp_d, const DriverOptions &driver
   uint32_t align_count = 0;
   uint32_t badp_count = 0;
 
-  zmq::message_t start_time(sizeof(stream_cmd.time_spec));
-  memcpy(start_time.data(), &stream_cmd.time_spec, sizeof(stream_cmd.time_spec));
-  start_trigger.send(start_time);
 
   zmq::message_t ring_size(sizeof(ringbuffer_size));
   memcpy(ring_size.data(), &ringbuffer_size, sizeof(ringbuffer_size));
   start_trigger.send(ring_size);
 
-
   //This loop receives 1 pulse sequence worth of samples.
+  auto first_time = true;
   while (1) {
     // 3.0 is the timeout in seconds for the recv call, arbitrary number
     size_t num_rx_samples = rx_stream->recv(buffer_ptrs, usrp_buffer_size, meta, 3.0);
+    if (first_time) {
+      std::cout << "rx meta " <<meta.time_spec.get_full_secs() << " " << meta.time_spec.get_frac_secs()<< std::endl;
```

Comment:
Hey! Looks like you're trying to compare time_spec from meta and stream_cmd, would you like help?
<img src="http://images.mentalfloss.com/sites/default/files/styles/mf_image_16x9/public/504767-Microsoft_0.jpg?itok=uMwRf0_2&resize=1100x619">
I suppose you could calculate the difference and print it out if there is a difference.